### DISCUSSION TITLE
imageCropper. How to set to image Blob object that stored in database.

### DISCUSSION THREAD

**Discussion by PeniukAndrii** [primefaces/community#2401] [PrimeFaces] [answered]
imageCropper. How to set to image Blog object that store in database. 

```xml
<p:imageCropper
	image="#{cvBean.image}"
	initialCoords="225,75,300,125"
	minSize="100,100"
	maxSize="250,250"
	zoomOnWheel="false"/>
```

```java
   private Blob image;

   public Blob getImage() {
          return image;
      }

//use it in init() method
  private void initImage() {
          image = service.getPhoto().getImage();
   }
```

I've tried to set StreamContent but it doesn't really work it doesn't show on page and in Network I saw this error
HTTP Status 404 – Not Found
Type Status Report
Message The requested resource [bla/bla/org.primefaces.model.DefaultStreamedContent@55b1d34c] is not available
Description The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.
Apache Tomcat/9.0.89  

Primefaces 8.0
Java 11

**Comment by melloware** [comment, marked as answer]
is your Bean ViewScoped?  You can't use StreamedContent with ViewScoped: https://primefaces.github.io/primefaces/15_0_0/#/core/dynamiccontent?id=viewscoped-support

Did you check the showcase it does exactly what you are doing and it works.

**Reply by PeniukAndrii** [reply]
I have @ManagedBean(name = "blabla") and @SessionScoped

**Reply by melloware** [reply]
ok check the showcase against what you are doing. its working fine in the Showcase

**Reply by PeniukAndrii** [reply]
ok, ty for answer
